api/client/v1beta1/conditions.go

```diff
@@ -0,0 +1,53 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	condition "github.com/openstack-k8s-operators/lib-common/modules/common/condition"
+)
+
+// OpenStackClient Condition Types used by API objects.
```

Comment:
the previous version contained `OpenStackControlPlane` Condition and Messages but these are not really used (I suspect that was unintentionally brought to infra-operator when we initially moved client cr from openstack-operator).

The latest version have these all removed. I didn't catch any problems in local sanity checks.